packages/monitoring-system/data-processor/README.md

```diff
@@ -30,91 +30,4 @@ Assign the Data Processor Service Account the role `Firebase Develop Admin` in t
 
 ### Firestore Schema
 
-The Data Processor requires the Firestore instance to have the following schema:
-
-```    
-"Bot": {                                         // collection
-    "[bot_id]": {                                // document
-    "bot_name": "[bot_name]"
-    }
-    ...
-},
-    
-"Task_Status_Queue": {                           // collection
-    "[bot_id]_[timestamp]": {                    // document
-    "in_queue": "[in_queue]"
-    }
-    ...
-},
-
-"Bot_Execution": {                               // collection
-    "[execution_id]": {                          // document
-    "bot_id": "[bot_id]",
-    "trigger_id": "[trigger_id]",
-    "start_time": "[start_time]",
-    "end_time": "[end_time]",
-    "logs_url": "[logs_url]"
-    }
-    ...
-},
-
-"Error": {                                       // collection
-    "[error_id]": {                              // document
-    "execution_id": "[execution_id]",
-    "error_msg": "[error_msg]",
-    "timestamp": "[timestamp]]",
-    "logs_url": "[logs_url]"
-    }
-    ...
-},
-
-"Trigger": {                                     // collection
-    "[trigger_id]": {                            // document
-    "github_event": "[github_event]",
-    "trigger_type": "[trigger_type]"
-    }
-    ...
-},
-
-
-"Action": {                                      // collection
-    "[action_id]": {                             // document
-    "execution_id": "[execution_id]",
-    "action_type": "[action_type]",
-    "destination_object": "[destination_object]",
-    "destination_repo": "[destination_repo]",
-    "value": "[value]",
-    "timestamp": "[timestamp]",
-    "logs_url": "[logs_url]"
-    }
-    ...
-},
-
-"Action_Type": {                                 // collection
-    "[action_type_id]": {                        // document
-    "description": "[description]",
-    "dev_hours_saved": "[dev_hours_saved]"
-    }
-    ...
-},
-
-"GitHub_Event": {                                // collection
-    "[payload_hash]": {                          // document
-    "repository": "[repository]",
-    "event_type": "[event_type]",
-    "timestamp": "[timestamp]",
-    "actor": "[actor]"
-    }
-    ...
-},
-
-"GitHub_Repository": {                           // collection
-    "[repo_name]_[owner_name]_[owner_type]": {}  // document
-    ...
-},
-
-"GitHub_Object": {                               // collection
-    "[object_type]_[repository]_[object_id]": {} // document
-    ...
-}
-```
\ No newline at end of file
+Refer to [firestore-schema.ts](packages/monitoring-system/data-processor/src/firestore-schema.ts) for a guide on the storage schema for Data Processor.
```

Comment:
The interface code is better documentation than a snippet in the README since you can see how the collections/documents fit together and it is guaranteed to be up-to-date